.github/workflows/unit_tests.yml

```diff
@@ -2,11 +2,7 @@ name: Unit tests
 
 on:
   push:
-    branches:
-      - master
   pull_request:
-    branches: 
-      - master
   schedule:
   - cron: "0 4 * * *"
```

Comment:
@alalazo Does the `cron` actually make sense? If so, why (and should we add a comment in the file to clarify)?

Reply:
OK, makes sense with that in mind, indeed.

So should we clarify that with a comment in `unit_tests.yml`?